content/environment_variables.tex

```diff
@@ -30,14 +30,29 @@
     \tabularnewline\hline
 %%
 \EnvVarDecl{SHMEM\_SYMMETRIC\_SIZE}
-    & Non-negative integer
-    & Number of bytes to allocate for symmetric heap
+    & Non-negative integer with optional character suffix
+    & Controls the size of the symmetric heap per \ac{PE}. The value
+      set in this environment variable is interpreted as number
+      of bytes, unless the number is followed by a single
+      character that acts as a multiplier, where:
+      \begin{itemize}
+        \item k or K multiplies by \(2^{10}\)  (kilobytes)
+        \item m or M multiplies by \(2^{20}\)  (megabytes)
+        \item g or G multiplies by \(2^{30}\)  (gigabytes)
+        \item t or T multiplies by \(2^{40}\)  (tebibytes)
```

Comment:
Use base-2 or base-10 size names consistently. "Tebibytes" is used here, while "gigabytes" et al. are used previously.